PluginCore/PluginCore/Helpers/PathHelper.cs

```diff
@@ -19,7 +20,7 @@ public static String BaseDir
         {
             get
             {
-                if (PluginBase.MainForm.StandaloneMode) return AppDir;
+                if (PluginBase.MainForm.StandaloneMode) return Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()));
```

Comment:
I understand the changes made to the path.
But it's just that the `$(BaseDir)` is defined as: `$(UserAppDir)` in normal cases, `$(AppDir)` in standalone mode.
The tests may not only use the base dir (although I think you've only changed base dir since it's the only one related to output path?) and in that case I think unsynchronised values could cause some unexpected results in weird places.